### DISCUSSION TITLE
HDD Speed test

### DISCUSSION THREAD

**Discussion by scubastan** [madMAx43v3r/chia-plotter#653] [Ideas]
Hey @madMAx43v3r 

How difficult would it be to create a HDD speed test program that simulates Chia Plotting?
I've tried fio, dd and the data I am getting is all over the place.  None of the results match the plot times i'm getting from ramdisk, sas ssd, sas hdd etc.

**Comment by gryan315** [comment]
Why not just benchmark by creating plots?
